features/other_a/invite_request.feature

```diff
@@ -156,3 +156,21 @@ Feature: Invite requests
     When I follow "Delete"
     Then I should see "Invitation successfully destroyed"
       And "user1" should have "4" invitations
+
+  Scenario: Translated email is sent when invitation request is declined by admin
+    Given a locale with translated emails
+      And invitations are required
+      And the user "user1" exists and is activated
+      And the user "notuser1" exists and is activated
+      And the user "user1" enables translated emails
+      And all emails have been delivered
+    When as "user1" I request some invites
+    When as "notuser1" I request some invites 
+    When I view requests as an admin
```

Comment:
We try to keep tests so that after `When` step, there are only `And` and `But` steps until the next `Then`. So please changes these to `And`.